app/modules/ads/app-configuration.js

```diff
@@ -0,0 +1,33 @@
+export default class AppConfiguration {
```

Comment:
I feel like I am missing something. I thought that kind of logic will be encapsulated within the ad-engine (hidden) so that client app doesn't know if it is using instant globals or instant config.

Reply:
It's a wrapper for combining instant-config and instant-globals. Once we switch fully to instant-config we'll be able to remove this class.